std/digest/crc.d

```diff
@@ -186,11 +188,11 @@ struct CRC32
             this = CRC32.init;
         }
         ///
-        unittest
+        @safe unittest
         {
             CRC32 digest;
             //digest.start(); //Not necessary
-            digest.put(0);
+            digest.put(ubyte(0));
```

Comment:
I can make it works but i'll be ugly. This case was only working with integer literals with a value less or equal to `ubyte.max`. The workaround to prevent the breaking change:

```D
void put(T)(T[] data...)
if (is(T == ubyte) || is(T == byte) || is(T == char)){}

void put(ubyte[] data...){put!ubyte(data);}

void main()
{
    put(0);
} 
```